crates/swc_ecma_utils/Cargo.toml

```diff
@@ -26,3 +26,4 @@ unicode-xid = "0.2"
 
 [dev-dependencies]
 testing = {version = "0.15.0", path = "../testing"}
+swc_ecma_parser = {path = "../swc_ecma_parser"}
```

Comment:
Can you specify the version?
Otherwise `cargo`will refuse to publish.

Reply:
Oh, I thought it would be ok because it's a dev dependency. I will update it.

Reply:
Fixed in a724e83efc275e72f47a024973d1ee948169af32.